tutorials/gSlack/index.md

```diff
@@ -0,0 +1,172 @@
+# gSlack
+
+This tutorial will explain how to get notified by [Slack](https://slack.com/) of various events that originate within [Google Cloud Platform](https://cloud.google.com/). `Google Compute Engine` instances may start and stop, new `Google AppEngine` verions might be deployed and `Google Cloud Storage` bucket may be created or deleted. People would like to be notified of these and others via their `Slack` account.
```

Comment:
Suggested edits:
`For example, `\`Google Compute Engine\` instances may start and stop, new \`Google AppEngine\` `versions` might be deployed and \`Google Cloud Storage\` bucket`s` may be created or deleted.